modules/install/pages/extensions.adoc

```diff
@@ -1,24 +1,316 @@
-= Extensions, Subsystems, and Profiles
+= Interpreting Extensions, Subsystems, and Profiles
 
-Brief blurb about the architecture of JBoss EAP (Do not repeat what will be already be covered in the Tech Overview course 1 in the path)
+* JBoss EAP 8.0 provides implementations of the Jakarta EE 10 APIs and Jakarta EE 10 https://jakarta.ee/specifications/webprofile/10/[Web Profile], https://jakarta.ee/specifications/coreprofile/10/[Core Profile], and the Full https://jakarta.ee/specifications/platform/10/[Platform] standards.
+* JBoss EAP 7.4, supported Jakarta EE 8. However, Jakarta EE 10 has undergone several changes, the most significant being the modification of the package namespace for Jakarta EE APIs from javax to jakarta.
+* See the complete list of specifications in the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/8.0/html-single/release_notes_for_red_hat_jboss_enterprise_application_platform_8.0/index#jakarta-ee-10-support_assembly-release-notes[release notes].
+
+== Modules in JBoss EAP
+
+* A module is a bundle composed of a library developed in Java and an XML configuration file that provides Jakarta Enterprise Edition (Jakarta EE) compliant functionalities.
```

Comment:
Using bundle confuses students - A library is a bundle of classes. Give a clear definition of modules in straight forward simple language.

Reply:
reframed and simplified